contracts/Market.sol

```diff
@@ -238,40 +242,42 @@ contract Market is Ownable, IMarket {
         return count; // token ID
     }
 
-    function settle(
-        uint256 index
-    ) external {
+    function settle(uint256 index) external {
         Bet memory bet = _bets[index];
         require(bet.settled == false, "settle: Bet has already settled");
-        bool result = IOracle(_oracle).checkResult(bet.marketId, bet.propositionId);
+        bool result = IOracle(_oracle).checkResult(
+            bet.marketId,
+            bet.propositionId
+        );
         _settle(index, result);
     }
 
-    function settleMarket(
-        uint256 from,
-        uint256 to,
-        bytes32 marketId
-    ) external {
-        for (uint256 i = from; i < to; i++) {
-            uint256 index = _marketBets[marketId][i];
-
-            if (!_bets[index].settled) {
-
-                bytes32 propositionId = IOracle(_oracle).getResult(_bets[index].marketId);
-
-                if (_bets[index].propositionId == propositionId) {
-                    _settle(index, true);
-                } else {
-                    _settle(index, false);
-                }
-            }
-        }
-    }
+    // function settleMarket(
```

Comment:
Are we no longer using this function, or will it be added back in the future?

Reply:
Talked to Barry, not gonna use for now, but not sure for the future, so I commented out.

Reply:
We should have settle market

Reply:
Talk to Barry again, we have ticket https://dltx.atlassian.net/jira/software/projects/HL/boards/82/backlog?selectedIssue=HL-153 to refactor the market contract, current settle market logic need to be change, I think we should comment out for now until we fixed the contract then redo unit test.